src/component/Collection/CollectionDetailItemComponent.js

```diff
@@ -57,16 +66,49 @@ export default class CollectionDetailItemComponent extends Component {
       return `선택된 트랙 (${ this.getTrackLength(trackList) })`;
     }
   }
-
-
+  
   handleSearchMapNameOnChange(e) {
     this.setState({ mapInputValue: e.target.value });
   }
 
+  handleMouseEnter(track) {
+    this.setState({ isShowPopup: true, selectedTrack: track });
+  }
+
+  handleMouseLeave() {
+    this.setState({ isShowPopup: false });
+  }
+
+  handleMouseMove(event) {
+    this.setState({
+      mouseX: event.clientX,
+      mouseY: event.clientY,
+    });
+  }
+
+  createPortalPopup() {
+    const { isShowPopup, selectedTrack, mouseX, mouseY } = this.state;
+    if (!isShowPopup) return null;
+      const style = {
```

Comment:
[Suggestion]
Is there a reason to use style here? If not, How about moving this to scss file? This repo tries to manage css and logic separately.